### PR TITLE
Patch to make multiqc compatible with version 2.0 of HiCExplorer

### PR DESCRIPTION
**Pull request by joachimwolff**
Hello Phil,

this is a patch for the HiCExplorer module, with version 2.0 we changed one string the QC log file. 

I have one important question: We plan to submit a paper (actually we have three weeks left to hand in) and for this we need the multiQC part working with HiCExplorer version 2.0. Can you tell me when do you plan to release the next version of multiQC? Or is it somehow possible to release a bug fixed version? 

Thanks a lot for your help,

Joachim

### CODE REVIEW

File: multiqc/modules/hicexplorer/hicexplorer.py [hunk 1]
```diff
@@ -51,17 +51,17 @@ def __init__(self):
...
         keys_list_read_orientation = ['Inward pairs', 'Outward pairs', 'Left pairs', 'Right pairs', 'Inter chromosomal']
 
         # prepare the detail report section
         self.add_section(
-            name = 'Mapping statistics',
-            anchor = 'hicexplorer_categorization_of_considered_reads',
-            plot = self.hicexplorer_create_plot(keys_categorization_of_reads_considered, 'HiCExplorer: Categorization of considered reads', 'categorization'),
-            description = 'This shows how the sequenced read pairs were mapped and those filtered due to mapping problems.',
-            helptext = '''
+            name='Mapping statistics',
```
Comment (ewels) [inline multiqc/modules/hicexplorer/hicexplorer.py]: Hah, style war! Apologies, I think it was me who went through adding spaces around all of these.

spaces

**Comment by ewels** [comment]
Looks good - I like to add a mention in the changelog but I can do this quickly.

I don't like to push releases _too_ frequently, thought it may be nice to do a new one fairly soon as I've recently patched bugs for updates in a few packages (eg. Picard which is obviously a common one). So I'll see what I can do.

Phil

**Comment by joachimwolff** [comment]
Thanks a lot. :+1:

File: multiqc/modules/hicexplorer/hicexplorer.py (hunk 1 above)
Comment (joachimwolff) [reply multiqc/modules/hicexplorer/hicexplorer.py]: I am always using my default automatic linter to lint it for me. My apologies if this is breaking any of your style guides.

Comment (ewels) [reply multiqc/modules/hicexplorer/hicexplorer.py]: It's no problem - I'm pretty relaxed when it comes to style (at least I think I am..?).

What's the linting tool out of curiosity?

Comment (joachimwolff) [reply multiqc/modules/hicexplorer/hicexplorer.py]: I am using autopep8 in combination with VS code. It's just a key command to let the linting run :)

**Pull request merged**